src/test/regress/expected/multi_alter_table_add_constraints.out

```diff
@@ -0,0 +1,501 @@
+ALTER TABLE products_ref DROP CONSTRAINT p_key;
+ALTER TABLE products_ref ADD CONSTRAINT p_key PRIMARY KEY(product_no);
+INSERT INTO products_ref VALUES(1, 'product_1', 1);
+-- Should error out, since we are trying to add new row having a value on p_key column
+-- conflicting with the existing row.
+INSERT INTO products_ref VALUES(1, 'product_1', 1);
+ERROR:  duplicate key value violates unique constraint "p_key_1400032"
+DETAIL:  Key (product_no)=(1) already exists.
+CONTEXT:  while executing command on localhost:57637
+DROP TABLE products_ref;
+-- Check "PRIMARY KEY CONSTRAINT" on append table
+CREATE TABLE products_append (
+    product_no integer,
+    name text,
+    price numeric
+);
+SELECT create_distributed_table('products_append', 'product_no', 'append');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- Cannot add primary key constraint on append table
+-- Command below should error out since 'name' is not a distribution column
+ALTER TABLE products_append ADD CONSTRAINT p_key PRIMARY KEY(name);
+ERROR:  cannot add UNIQUE, PRIMARY KEY or EXCLUDE constraint on "products_append" 
+DETAIL:  UNIQUE constraints, EXCLUDE constraints, and PRIMARY KEYs on append-partitioned tables cannot be enforced.
+HINT:  Consider using hash partitioning.
+ALTER TABLE products_append ADD CONSTRAINT p_key PRIMARY KEY(product_no);
+ERROR:  cannot add UNIQUE, PRIMARY KEY or EXCLUDE constraint on "products_append" 
+DETAIL:  UNIQUE constraints, EXCLUDE constraints, and PRIMARY KEYs on append-partitioned tables cannot be enforced.
+HINT:  Consider using hash partitioning.
+COPY products_append FROM STDIN DELIMITER AS ',';
+DROP TABLE products_append;
+-- Check "UNIQUE CONSTRAINT"
+CREATE TABLE unique_test_table(id int, name varchar(20));
+SELECT create_distributed_table('unique_test_table', 'id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- Can only add unique constraint on distribution column (or group
+-- of columns including distribution column)
+ALTER TABLE unique_test_table ADD CONSTRAINT unn_name UNIQUE(name);
+ERROR:  cannot create constraint on "unique_test_table"
+DETAIL:  Distributed relations cannot have UNIQUE, EXCLUDE, or PRIMARY KEY constraints that do not include the partition column (with an equality operator if EXCLUDE).
+ALTER TABLE unique_test_table ADD CONSTRAINT unn_id UNIQUE(id);
+-- Error out from the related shard. Table can not have two rows with same id.
+INSERT INTO unique_test_table VALUES(1, 'Ahmet');
+INSERT INTO unique_test_table VALUES(1, 'Mehmet');
+ERROR:  duplicate key value violates unique constraint "unn_id_1400035"
+DETAIL:  Key (id)=(1) already exists.
+CONTEXT:  while executing command on localhost:57638
+ALTER TABLE unique_test_table DROP CONSTRAINT unn_id;
+-- Can create unique constraint over multiple columns which must include
+-- distribution column
+INSERT INTO unique_test_table VALUES(1, 'Mehmet');
```

Comment:
According to comment, I expect that next example will be successfully adding a unique constraints over multiple columns, but we test failing case when there is already conflicting rows in the table. Let's keep both tests, but write separate comments for them.